misc/plot-roh.py

```diff
@@ -352,6 +359,16 @@ def parse_samples(fname,highlight):
 off = 0
 off_sep = 0
 dat_rg1 = {}
+
+if not chrs:
+    print("No GT tags found in input file.\n\nPlease try rerunning using run-roh.pl with --roh-args \"-G 30\"")
+    print("\nAlternatively run these commands manually:")
+    print("bcftools annotate --rename-chrs chr-names.txt in.bcf -Ob -o annotated.bcf")
```

Comment:
This is not something to be applied blindly, even though run-roh.pl does it (the script is a bit simplistic in this sense). This would be used only when the chromosome naming convention is different in the genetic map file or in the AF file. However, as it's not used anywhere in the commands below, it does not need to be listed here. Only the `query` command is essential - whichever way the user produced the roh output, now only the GT lines need to be added.

Reply:
So, should I just take this bit out?

Reply:
Yeap :+1:

Reply:
... and done.